core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -1205,6 +1210,13 @@ object Chunk {
      * Zips this chunk with the specified chunk using the specified combiner.
      */
     override def zipWith[B, C](that: Chunk[B])(f: (Nothing, B) => C): Chunk[C] = Empty
+
+    //override type Concat[A1] = Chunk[A1]
+
+    /**
+     * Returns the concatenation of this chunk with the specified chunk.
+     */
+    def ++[A1](that: Chunk[A1]): Chunk[A1] = that
```

Comment:
I think there is some duplication between this and the implicit syntax class we can get rid of.